chromium_src/chrome/browser/prefs/browser_prefs.cc

```diff
@@ -13,6 +13,10 @@
 #include "brave/browser/widevine/widevine_utils.h"
 #endif
 
+#if !defined(OS_ANDROID)
```

Comment:
This file seems to be inconsistently guarded. When it's used in so many places I think it's generally better to guard it internally with no-op in android rather that try to guard it in a dozen different places

Reply:
Yup, made this file no-op for android.